apple/REAUIKit.h

```diff
@@ -0,0 +1,21 @@
+#if !TARGET_OS_OSX
+
+#import <UIKit/UIKit.h>
+
+#define REAUIView UIView
```

Comment:
Note: We have a desire to convert all our '#define's in React Native macOS to typedefs, so as to move the check to compile time where the IDE can find more errors. If this was inspired by our repo, know that I'm probably going to change it eventually :D

Reply:
@Saadnajmi Thanks for pointing this out! Do you think we can use typedefs now or should we wait for the changes to appear in react-native-macos first?

Reply:
I think you want to have your own type? Otherwise your iOS code will depend on React Native macOS if I'm not mistaken?